lib/galaxy/queue_worker.py

```diff
@@ -18,6 +19,21 @@
 log = logging.getLogger(__name__)
 
 
+def queue_async_task(app, task, kwargs={}):
+    log.info("Queuing async task %s." % task)
```

Comment:
I think the name and lack of docstring in this and `send_control_task` was confusing me for a moment:
The difference between `send_control_task` and `queue_async_task` is that `queue_async_task` is consumed just once by the same process, right? I'm asking because `send_control_task` also queues async tasks, with the difference that they are consumed by all processes (or all except the current process). Maybe we could call this `send_local_control_task` ?